lib/Common/sensors_state.h

```diff
@@ -32,6 +31,7 @@ struct SensorStateSnapshot {
   bool brewActive;
   bool steamActive;
   bool hotWaterSwitchState;
+  bool tarePending;
```

Comment:
This shouldn't be here but in SystemState instead right? (`system_state.h` file)

Reply:
shut yeah right forgot this one